pyproject.toml

```
@@ -42,7 +42,7 @@ preql = {version="^0.2.19", optional=true}
 vertica-python = {version="*", optional=true}
 urllib3 = "<2"
 oracledb = {version = "*", optional=true}
-pyodbc = {version="^4.0.39", optional=true}
+pyodbc = {version=">=4.0.39", optional=true}
```

```suggestion
pyodbc = {version="^4.0.39", optional=true}
```
We only install minor patches to prevent breaking changes. If you have a compelling reason to update this, open a separate PR.

---

The dependency is related to MSSQL and MSSQL is not supported natively. That's why I decided to update the same PR. The right thing to do would be to remove all MSSQL-related files.

---

We support MSSQL in Datafold Cloud so some dependencies are inherited from this repo. Once you commit the suggestion, I'll approve and merge!